### DISCUSSION TITLE
Attributt: Helsepersonellets fødselsnummer og navn

### DISCUSSION THREAD

**Discussion by steinarnoem** [NorskHelsenett/Tillitsrammeverk#6] [General]
Skal både fødselsnummer og navn være med i datamodellen, eller holder det med fødselsnummer?
En fordel med å ta med begge deler er at datakilden unngår å gjøre oppslag selv..

**Comment by micer2004** [comment]
Tenker at de burde holde med personidentifikator i form av F- eller D-nummer. Gjør ikke HelseID en form for verifisering mot sentrale kilder som er bl.a. Folkeregisteret om at den som autentiseres er den som den påstår? Man kan evt. berike med folkeregistrert navn som en del av  denne operasjonen. Hva tenker du, @steinarnoem ?

**Reply by AMJohnsen** [reply]
Kan det være en tanke at NIN oppslag gjøres hos virksomheten? Verifiseringen kan i prinsippet gjøres hos kilde, helseID eller konsument. Det bør gås opp :)

**Comment by mortsten** [comment]
Da navn er sentralt i videre bruk, f.eks logg, tenker jeg at det bør være med, men kan jo også løses med oppslag der behovet oppstår

**Comment by stvgr** [comment]
Stavanger mener at det viktigste er at innbygger ser navnet når hen leser sin logg, og at systemadministrator ser navnet når hen leser i kommunens logg.

**Comment by richardhus** [comment]
Helse Sør-Øst har også lagt til grunn at navn medfølger for oppsøkende helsepersonell, slik at dette kan logges. Tilbakeoppslag i folkeregisteret for å hente navnet høres unødvendig ut fra kildenes side i alle fall. Det gir også sporbarhet i loggene at det ikke bare er en identifikator som peker på et navn som over tid kan endres i folkeregisteret. Innbyggere vil ikke nødvendigvis kjenne igjen personer som senere har byttet navn når de ser på sin tilgangslogg, og dokumentasjonen fra samme tidspunkt vil kanskje inneholde tidligere navn fremfor siste navn. Og trenger man å vite nåværende navn er det sikkert enklere å spore opp basert på tidligere navn enn motsatt vei?

**Reply by micer2004** [reply]
Godt poeng med at en person kan bytte navn som resultat av giftemål eller noe annet. Navnet bør foreligge statisk i de loggene og ikke endre seg over tid. Samtidig er det behov for en norsk identifikator for norsk helsepersonell som bør også persisteres, i tillegg til navnet, slik at ved behov kan man hente nyeste opplysninger fra Folkeregisteret, HPR eller andre tjenester som har fokus på historisk perspektiv.
